presto-hive/src/main/java/com/facebook/presto/hive/HiveSessionProperties.java

```diff
@@ -115,6 +116,7 @@
     public static final String FILE_RENAMING_ENABLED = "file_renaming_enabled";
     public static final String PREFER_MANIFESTS_TO_LIST_FILES = "prefer_manifests_to_list_files";
     public static final String MANIFEST_VERIFICATION_ENABLED = "manifest_verification_enabled";
+    public static final String METASTORE_USER_SUPPLIED_EXTRA_PARAM = "metastore_user_supplied_extra_param";
```

Comment:
Currently the extra parameters are only set for new partitions. What do you think about reflecting it in the session property name. 

For example we can define the session property to be a map (similarly to https://github.com/prestodb/presto/blob/883e88322b229b01a790c12a37fd2ee2ab51b26e/presto-hive/src/main/java/com/facebook/presto/hive/HiveTableProperties.java#L82) and call it `(NEW_|CREATE_|)PARTITION_EXTRA_PARAMETERS`.

Reply:
Yeah I think the name has room for improvement.

I don't want to make it a map because I don't want to dictate the structure of the value people put in it.